tests/playwright/pages/wp-admin-page.js

```diff
@@ -73,15 +74,67 @@ module.exports = class WpAdminPage extends BasePage {
 	}
 
 	/**
-	 * Determine which experiments are active / inactive.
+	 * Activate / Deactivate Experiments.
 	 *
 	 * TODO: The testing environment isn't clean between tests - Use with caution!
 	 *
 	 * @param {Object} experiments - Experiments settings ( `{ experiment_id: true / false }` );
+	 * @param {string} mode        - cli (default) or ui
 	 *
 	 * @return {Promise<void>}
 	 */
-	async setExperiments( experiments = {} ) {
+	async setExperiments( experiments = {}, mode = 'cli' ) {
+		if ( 'ui' === mode.toLowerCase() ) {
+			await this._setExperimentsInUI( experiments );
+			return;
+		}
+
+		await this._setExperimentsInCLI( experiments );
+	}
+
+	/**
+	 * Set experiments using the CLI.
+	 *
+	 * @param {Object} experiments - Experiments settings ( `{ experiment_id: true / false }` );
+	 */
+	async _setExperimentsInCLI( experiments = {} ) {
+		let experimentsToEnable = [],
+			experimentsToEnableString,
+			experimentsToDisable = [],
+			experimentsToDisableString;
+
+		await parseExperiments();
+		const cli = new Terminal;
+
+		if ( experimentsToEnableString ) {
+			await cli.experiments( 'activate', experimentsToEnableString );
+		}
+		if ( experimentsToDisableString ) {
+			await cli.experiments( 'deactivate', experimentsToDisableString );
+		}
+
+		async function parseExperiments() {
+			for ( const [ id, state ] of Object.entries( experiments ) ) {
+				switch ( true ) {
+					case 'inactive' === state || 'false' === state:
```

Comment:
Can the state be true if the experience is inactive?

Reply:
Experiments are passed to setExperiment() with true/false active/inactive values in our existing code. I've extended this interface to the cli. By doing this we the addition does not require changing how we call setExperiments()

```
await wpAdmin.setExperiments( {
		e_swiper_latest: false,
} );

await wpAdmin.setExperiments( {
	container: 'active',
	'nested-elements': 'active',
} );